h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
```

Comment:
Agree completely, although I'm not sure "SessionService" makes a lot of sense -- it's not really got anything to do with the session these days.